File: Smodr/src/main/java/com/kevintcoughlin/smodr/utils/StringResourceUtilities.java
package com.kevintcoughlin.smodr.utils;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.annotation.StringRes;

import com.google.android.material.snackbar.Snackbar;

public final class StringResourceUtilities {
    public static String getString(@Nullable final Context context, @StringRes final int id, Object... args) {
        return context != null ? context.getResources().getString(id, args) : "";
    }
}

File: Smodr/src/main/java/com/cascadiacollections/smodr/utils/StringResourceUtilities.kt
package com.cascadiacollections.smodr.utils

import android.content.Context
import androidx.annotation.StringRes

object StringResourceUtilities {
    @JvmStatic
    fun getString(context: Context?, @StringRes id: Int, vararg args: Any?): String {
        return context?.resources?.getString(id, *args) ?: ""
    }
}